frontend/src/components/search/components/common/filter-select/filter-select.tsx

```diff
@@ -0,0 +1,54 @@
+import { FC } from 'common/types/types';
+import { useState } from 'hooks/hooks';
+import { IconName } from 'common/enums/components';
+import { Icon } from 'components/common/common';
+
+import styles from './styles.module.scss';
+
+type Props = {
+  filter: string;
+  onChangeFilter: (value: string) => void;
+  title: string;
+  options: {
+    text: string;
+    value: string;
+  }[];
+};
+
+const FilterSelect: FC<Props> = ({ filter, onChangeFilter, title, options }) => {
+  const [filterTitle, setFilterTitle] = useState(title);
+
+  const onHandleChangeFilter = (e: React.FormEvent<HTMLInputElement>): void => {
+    onChangeFilter(e.currentTarget.value);
+    setFilterTitle(e.currentTarget.dataset.name as string);
+  };
+
+  return (
+    <>
+      <div className={styles['filter-select']}>
+        <div className={styles['filter-select-title']} tabIndex={13}>
+          {filterTitle}
+          <Icon name={IconName.ARROW_DOWN} color="gray" />
+        </div>
+        <div className={styles['filter-select-list']}>
+          {options.map((o) => (
+            <div key={o.value} className={styles['filter-select-item']}>
+              <input
+                id={`radS-${o.value}`}
+                type="radio"
+                name={title}
+                value={o.value}
+                data-name={o.text}
+                checked={filter === o.value}
+                onChange={onHandleChangeFilter}
+              />
+              <label htmlFor={`radS-${o.value}`}>{o.text}</label>
+            </div>
+          ))}
+        </div>
+      </div>
+    </>
+  );
+};
+
+export { FilterSelect };
```

Comment:
I saw above mostly the same component. Can't you reuse one component?

Reply:
@kyrylo-burmelov If you mean 'filter-select' component, it is realy similar, but has another styles to use like standart select.